tests/devices/i19/test_diffractometer.py

```diff
@@ -0,0 +1,7 @@
+from dodal.devices.i19.diffractometer import DetectorMotion, FourCircleDiffractometer
+
+
+def test_diffractometer_created_without_errors():
```

Comment:
Could: Might be good to test that the readables are as expected. You can do this with `assert_reading`.